docker-run.sh

```diff
@@ -6,6 +6,7 @@ docker run -it \
   -p 1234:1234 \
   -p 5678:5678 \
   -p 4000:4000 \
+  -p 5000:5000 \
```

Comment:
I thought I was able to access the public UI via http://localhost:5678, but maybe I'm mistaken.

Reply:
It appeared as a loading indicator indefinitely, since the backend service couldn’t be reached. Confirmed with Eric that this port should be accessible for the service — if you find that you’re able to run this locally without it, there might be something fishy up!

Reply:
ah ok, I do get the load page. good to know!